__std__/nix/doc/scripts/sphinx-autobuild-doc-site.nix

```diff
@@ -0,0 +1,13 @@
+{ inputs, cell }:
+
+inputs.nixpkgs.writeShellApplication {
+  name = "autobuild";
```

Comment:
Per the policy, shouldn't this be the same as the file name?

Reply:
This is actually the executable name, which is one layer below, if you will.
The reason why I chose a short name is that these `scripts` end up in the devshell as commands available in the terminal. 
The idea is that you might want to type these very frequently, and therefore it's convenient for them to require few keystrokes.
For example, when entering the `doc-shell`, you are presented with this menu 
```
[doc]

  autobuild       - Start the autobuild server in doc/_build
  build           - Build the docs locally in doc/_build
  nix-serve       - Full nix build of the doc + serve them on port 3000

[general commands]

  check-the-flake - builds everything in the flake
  menu            - prints this menu
  repo-root       - prints the repository root path
  wip             - alias for: git add . && git commit -m WIP
  ```
  
  And that's where the `autobuild` comes from.

Reply:
Totally makes sense. Then you should update your policy regarding this in `flake.nix` (not *all* derivation need to have the same name as the filename).

Reply:
Almost: see this https://github.com/input-output-hk/plutus/pull/4832#discussion_r959519286